python_modules/dagster/dagster_tests/core_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -62,21 +65,32 @@
     migrate_asset_key_data,
 )
 from dagster.core.storage.event_log.sqlite.sqlite_event_log import SqliteEventLogStorage
-from dagster.core.test_utils import instance_for_test
+from dagster.core.test_utils import create_run_for_test, instance_for_test
 from dagster.core.utils import make_new_run_id
 from dagster.loggers import colored_console_logger
 from dagster.serdes import deserialize_json_to_dagster_namedtuple
 from dagster.utils import datetime_as_float
 
-DEFAULT_RUN_ID = "foo"
-
 TEST_TIMEOUT = 5
 
 # py36 & 37 list.append not hashable
 # pylint: disable=unnecessary-lambda
 
 
-def create_test_event_log_record(message: str, run_id: str = DEFAULT_RUN_ID):
+@contextmanager
+def create_and_delete_test_runs(instance: DagsterInstance, run_ids: List[str]):
+    check.inst_param(instance, "instance", DagsterInstance)
+    check.list_param(run_ids, "run_ids", of_type=str)
+    if instance:
+        for run_id in run_ids:
+            create_run_for_test(instance, run_id=run_id)
+    yield
+    if instance:
+        for run_id in run_ids:
+            instance.delete_run(run_id)
+
+
+def create_test_event_log_record(message: str, run_id: str = "foo"):
```

Comment:
I think we're still relying on the event log record's run_id and the default_run_id param being `foo`?

Do we save a lot of boilerplate in having the `default_run_id` fixture rather than creating a run in the `_synthesize_events` call?